src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/paymentFromComponent.js

```diff
@@ -56,6 +74,10 @@ function paymentFromComponent(req, res, next) {
     });
   });
 
+  if (AdyenHelper.isApplePay(reqDataObj.paymentMethod.type)) {
+    result = setApplePayData(result, order);
```

Comment:
This function manipulates the input `result` rather than returning a new `result` object